### ISSUE TITLE
Create an activity if an object is deleted

### ISSUE THREAD

**Issue by daria-sl** [globe-swiss/phaenonet-client#47]
If an object is deleted and someone follows this object or the person it belongs to, an activity should be created. @daria-sl : please define layout and content of this activity.

This issue is not urgent.

**Comment by EricWyss64** [comment]
Dieser Change kann weggelassen werden
